packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/delete-token/delete-token.routing.js

```diff
@@ -0,0 +1,25 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.ai-dashboard.users-tokens.delete-token',
+    {
+      url: '/delete-token',
```

Comment:
**nitpick:** `token` param needs to be included in the URL to have a consistent behaviour on refresh of Delete token page.

Reply:
Done here: https://github.com/ovh/manager/pull/8443/commits/2c7a7ce3dcf42f27918184c2a4679322dd24117d